Libraries/LibCrypto/Curves/SECPxxxr1.h

```diff
@@ -99,7 +99,7 @@ class SECPxxxr1 : public EllipticCurve {
 
     // Curve parameters
     static constexpr size_t KEY_BIT_SIZE = bit_size;
-    static constexpr size_t KEY_BYTE_SIZE = KEY_BIT_SIZE / 8;
+    static constexpr size_t KEY_BYTE_SIZE = byte_size;
```

Comment:
Why do this instead of `KEY_BYTE_SIZE = ceil_div(bit_size, 8)`?

Reply:
Yeah that's easier, thank you for the pointer!